README.md

```diff
@@ -185,7 +185,7 @@ with ENV:
  * `AutomaticEnv()`
  * `BindEnv(string...) : error`
  * `SetEnvPrefix(string)`
- * `SetEnvKeyReplacer(string...) *strings.Replacer`
+ * `SetEnvKeyReplacer(string...) *replacer`
```

Comment:
I think this should be `SetEnvKeyReplacer(r replacer)`, since the replacer is not a return value.